app/src/main/java/org/mozilla/fenix/nimbus/NimbusBranchesFragment.kt

```diff
@@ -30,65 +52,145 @@ import org.mozilla.fenix.nimbus.view.NimbusBranchesView
 class NimbusBranchesFragment : Fragment() {
 
     private lateinit var nimbusBranchesStore: NimbusBranchesStore
-    private lateinit var nimbusBranchesView: NimbusBranchesView
-    private lateinit var controller: NimbusBranchesController
 
     private val args by navArgs<NimbusBranchesFragmentArgs>()
 
+    override fun onResume() {
```

Comment:
Could you undo the move of `onResume` and `loadExperimentBranches`? Since both these functions didn't actually change and were only moved above `onCreateView`, we would prefer to minimize the amount of changes in the PR.